bindep.txt

```diff
@@ -22,7 +22,6 @@ podman            [platform:rpm]
 python3-devel     [platform:rpm !platform:rhel-7 !platform:centos-7]
 python3-libvirt   [platform:rpm]
 python3-lxml      [platform:rpm]
-PyYAML            [platform:rpm !platform:rhel-8 !platform:centos-8 !platform:rhel-9 !platform:centos-9 !platform:fedora]
```

Comment:
why not add `platform:centos-10` ?

```
podman run -it quay.io/centos/centos:stream10 bash 
yum install -y git vim python3-pip make sudo
git clone https://github.com/openstack-k8s-operators/ci-framework && cd ci-framework
./scripts/setup_env


[root@f210a56a991a ci-framework]# bindep --profiles
Platform profiles:
platform:base-py3
platform:centos
platform:centos-10
platform:centos-coughlan
platform:redhat
platform:rpm
```

Reply:
[platform:rpm !platform:rhel-8 !platform:centos-8 !platform:rhel-9 !platform:centos-9 !platform:fedora]

becaues the condtion say

install on rpm distros that are not one of rhel 8, rhel 9, centos 8 centos 9 or fedora

this repo does not support centos/rhel7  and the conditions have eliminated all but centos/rhel10  and that is what we would be adding.

so i suggested removing the package entrily since we have eliminated all distos it could be installed on with the conditions.